test/integration/puppetlabs/services/jruby/jruby_puppet_pool_int_test.clj

```diff
@@ -284,7 +267,7 @@
            (tk-app/stop app)))))))
 
 (deftest ^:integration test-503-when-jruby-is-first-to-shutdown
-  (testing "During a shutdown requests result in 503 http responses"
+  (testing "After a shutdown, requests result in 503 http responses"
```

Comment:
I'm having a hard time understanding what exactly the difference is between this test and the test above (`test-503-when-app-shuts-down`). It seems like the difference now is that in the first we shut down the whole app, but in this one we just shut down the `JRubyService`? Maybe it would be good to add a little bit more to the testing strings or comments to distinguish them? (I guess this isn't really a comment entirely about this PR...)